include/scalarfield.tcc

```diff
@@ -29,71 +29,76 @@
 #ifndef SMARTREDIS_SCALARFIELD_TCC
 #define SMARTREDIS_SCALARFIELD_TCC
 
+// ScalarField constructor
 template <class T>
-ScalarField<T>::ScalarField(const std::string& name,
-                            MetaDataType type) :
+ScalarField<T>::ScalarField(const std::string& name, MetaDataType type) :
```

Comment:
Not a big deal, but it looks like in other functions below you put the ``:`` on the next line.  Either is fine, probably just better to be consistent

Reply:
I tried to get them all moved to the next line to match the style recommendation from the Google style docs. Thanks for pointing out cases where I missed it